scripts/appbuilder

```diff
@@ -141,45 +142,64 @@ fi
 docker_network_remove()
 {
     i=0
-    until [[ $i -ge $DOCKER_RETRIES ]]
-    do
+    until [[ ${i} -ge ${DOCKER_RETRIES} ]]; do
         # Try to remove docker network
-        if (docker network remove $NETWORK_NAME)
-        then
+        if (docker network remove ${NETWORK_NAME}); then
             rmerrexit=$?
             break
         else
             rmerrexit=$?
             i=$((i+1))
         fi
         # Handle failed removal
-        if [[ $rmerrexit -ne 0 ]] && [[ $i -ge $DOCKER_RETRIES ]]
-        then
+        if [[ ${rmerrexit} -ne 0 ]] && [[ ${i} -ge ${DOCKER_RETRIES} ]]; then
             # Exit with error
-            echo "Failed to remove Docker network $NETWORK_NAME"
-            if (docker network ls | grep $NETWORK_NAME)
-            then
+            echo "Failed to remove Docker network ${NETWORK_NAME}"
+            if (docker network ls | grep ${NETWORK_NAME}); then
                 # Print out network information for debugging
-                docker network inspect $NETWORK_NAME
+                docker network inspect ${NETWORK_NAME}
             fi
-            exit $rmerrexit
+            exit ${rmerrexit}
         else
             # Attempt to resolve and retry removal
-            if (docker network ls | grep $NETWORK_NAME >> /dev/null)
-            then
+            if (docker network ls | grep ${NETWORK_NAME} >> /dev/null); then
                 # Force disconnect any containers still attached to network
-                for CONTAINER in $(docker network inspect --format '{{range $k, $v := .Containers}}{{print $k}}{{end}}' $NETWORK_NAME)
-                do
-                    echo "Force removing $CONTAINER from $NETWORK_NAME"
-                    docker network disconnect -f $NETWORK_NAME $CONTAINER;
-                done;
+                for CONTAINER in $(docker network inspect --format '{{range $k, $v := .Containers}}{{print $k}}{{end}}' ${NETWORK_NAME}); do
+                    echo "Force removing ${CONTAINER} from ${NETWORK_NAME}"
+                    docker network disconnect -f ${NETWORK_NAME} ${CONTAINER} || true
+                done
             fi
-            echo "Docker network retry $i: sleeping $i seconds..."
+            echo "Docker network $(basename ${PWD}) retry ${i}: sleeping ${i} seconds..."
             sleep $i
         fi
     done
 }
 
+docker_build()
```

Comment:
It might be possible to reuse `scripts/retry_w_backoff` here.

Reply:
Thanks for this suggestion VIkas.

Reply:
Thanks for this suggestion VIkas.